snippets/list-entities-query-fields.mdx

```diff
@@ -18,8 +17,218 @@ export const listAccountsQueryFields = {
         {name:"primary_email",type: "string"}
     ],
     sort_fields: [
-        {name:"account_id",type: "string", description: ""},
-        {name:"updated_at",type: "string", description: ""}
+        {name:"account_id",type: "string", description: "Unique identifier of the account"},
+        {name:"updated_at",type: "string", description: "Last updated time of the record"}
```

Comment:
I didn't add description to fields in sort_fields earlier because mostly the field will be repeated from query_fields.